app/models/line_item.rb

```diff
@@ -44,6 +44,8 @@ def type
 
   def cost=(value)
     self[:cost] = BigDecimal((value || 0).to_s).round(DECIMALS)
+  rescue ArgumentError
```

Comment:
so $5.50 is a problem (for example) ?

Reply:
yes @macejmic because there is another attribute called `currency` that is the one for the dolar. `cost` is only the number/value.